composer.json

```diff
@@ -11,14 +11,14 @@
         }
     ],
     "require": {
-        "php": "^5.5|^7.0"
+        "php": "^5.5 || ^7.0"
     },
     "require-dev": {
         "php-http/httplug": "^1.0",
         "php-http/message-factory": "^1.0",
         "puli/composer-plugin": "1.0.0-beta9",
```

Comment:
If our strategy works with beta9 sure. We support it. (I think it does)

Im not sure `^1.0.0-beta9` is the same as `1.0.0-beta9 || 1.0.0-beta10`. And if it is, we should not use it. Because we can guarantee it works with `1.0.0-beta11`